main.qml

```diff
@@ -104,6 +104,11 @@ ApplicationWindow {
         else if(seq === "Ctrl+I") middlePanel.state = "Sign"
         else if(seq === "Ctrl+E") middlePanel.state = "Settings"
         else if(seq === "Ctrl+D") middlePanel.state = "Advanced"
+        else if(seq === "Ctrl+Y") {
+            leftPanel.selectItem("Keys")
+            settingsPasswordDialog.open()
```

Comment:
- Why not doing `middlePanel.state = "Keys"` like other tabs?

- I did some refactoring on the password dialog logic in #889, so this line needs to be updated accordingly.